R/demo.Rmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "demo_plot_forecast"
+author: "Serena Wang"
+date: "10/19/2020"
+output: html_document
+---
+
+```{r message=FALSE}
+library(zoltr)
+library(dplyr)
+library(ggplot2)
+library(tidyverse)
+library(covidHubUtils)
+```
+
+```{r message=FALSE}
+# Load forecasts that were submitted in a time window from zoltar
+inc_case_targets <- paste(1:4, "wk ahead inc case")
+forecasts <- covidHubUtils::load_forecasts(models = "COVIDhub-ensemble",
+                      last_forecast_date = Sys.Date(),
+                      forecast_date_window_size = 8,
+                      locations = "US",
+                      types = c("point", "quantile"),
+                      targets = c(inc_case_targets),
+                      source = "zoltar",
+                      hub_repo_path = "")
+
+```
+
+```{r message=FALSE}
+# Plot point forecasts with prediction intervals
+covidHubUtils::plot_forecast (forecast_data = forecasts,
```

Comment:
this is great. could we make the options for model, location optional IF only one model and location are in the forecast_data?
And have horizon and quantile default to include what is present in the data which could be overwritten by specification?
I don't think this should hold up merging this PR (we could create a separate issue to track this).